STORM-IRIT/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM ubuntu:18.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+RUN apt-get install -y cmake pkg-config  wget unzip
+
+#CGAL installation:
+RUN apt-get install -y libgmp-dev libmpfr-dev libboost-all-dev
+
+WORKDIR /root
+
+RUN wget https://github.com/CGAL/cgal/releases/download/releases%2FCGAL-5.0.2/CGAL-5.0.2.zip
+
+RUN unzip CGAL-5.0.2.zip
+
+RUN rm CGAL-5.0.2.zip
+
+RUN mv CGAL-5.0.2 CGAL
+
+WORKDIR /root/CGAL/build
+
+RUN cmake .. && make
+
+ENV CGAL_DIR "/root/CGAL"
+
+#Building STORM IRIT package
+ADD ./src /app/src/src
```

Comment:
You can use regexps (resolved by Go's match) to add all files in one ADD instead of 3